src/main/java/com/yelp/nrtsearch/server/luceneserver/SearchHandler.java

```diff
@@ -177,9 +184,11 @@ public SearchResponse handle(IndexState indexState, SearchRequest searchRequest)
       }
 
       // add results from any extra collectors
-      searchContext
-          .getResponseBuilder()
-          .putAllCollectorResults(searcherResult.getCollectorResults());
+      if (searcherResult != null) {
```

Comment:
This is the only difference with the typical search functionality - we do not have a `searcherResult` object